tests/xbot/common/command/scripted/ScriptedCommandTest.java

```diff
@@ -0,0 +1,118 @@
+package xbot.common.command.scripted;
+
+import static org.junit.Assert.*;
+
+import org.apache.log4j.Logger;
+import org.junit.Test;
+
+import xbot.common.command.scripted.ScriptedCommand;
+
+public class ScriptedCommandTest extends BaseScriptedCommandTest {
```

Comment:
You are missing a few tests:
-You need a test that loads a script from a file - that's how the robot is going to do it, so you want to make sure that path is solid.
-One where the file does not exist
-One while the script calls a command that does not exist
-One where the file/script is empty/null string
-One where your scripted command is interrupted while some work is going on underneath it